docs/docs/sql-syntax/functions/geometry/get-x.md

```diff
@@ -1 +1,53 @@
-# GET_X
\ No newline at end of file
+# GET_X
+
+The `GET_X` function returns the x-coordinate of a given `POINT` data type. It takes one `POINT` data type argument and returns a `FLOAT` value representing the x-coordinate.
+
+## Syntax
+
+```sql
+GET_X(point)
+```
+
+## Examples
+
+Create a table with a default value for the `id` column using `GET_X`:
+
+```sql
+CREATE TABLE SingleItem (id FLOAT DEFAULT GET_X(POINT(0.3134, 0.156)));
+```
+
+Get the x-coordinate of a point value:
+
+```sql
+SELECT GET_X(CAST('POINT(0.1 -0.2)' AS POINT)) AS ptx;
+```
+
+```sql
+SELECT GET_X(POINT(0.1, -0.2)) AS ptx;
+```
+
+Create a table with a `POINT` data type column:
+
+```sql
+CREATE TABLE POINT (point_field POINT);
```

Comment:
Consider changing the POINT table that uses UPPERCASE to PascalCase, as it can be mistaken with the DataType POINT.